### DISCUSSION TITLE
Deployment to the website

### DISCUSSION THREAD

**Discussion by FAJ-Munich** [DivinumOfficium/divinum-officium#4126] [Developers corner]
Maybe a superfluous question:
I experience that deployment to the website appears to be "jumping", i.e., on multiple re-loads in the browser the result is randomly chosen between an older version (without the latest commits) and the latest version (including the latest commits). I am not sure if there is a way around such behaviour: certainly, it matters the most when bug fixes for the code are deployed.

**Comment by APMarcello3** [comment]
@benyanke

**Comment by benyanke** [comment]
Good afternoon @FAJ-Munich  - this is expected behavior during deployment, as our deployment utilizes a rolling no-downtime deployment. It is only expected to occur for a short time during the time a deployment is occurring. 

During the short deployment window, it is possible that some HTTP requests are responded to from an old instance with the previous commit, and some are responded to with the latest commit. This is expected behavior during deployments and not an issue.

**Reply by FAJ-Munich** [reply]
Thanks for the explanation! Highly appreciated. 
Laudetur Jesus Christus!

**Reply by benyanke** [reply]
You're welcome. Thank you for the report!
